CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Upcoming
 
+- Changed type generation for `grpcio` stubs to use the `MultiCallable` API ([see here](https://grpc.github.io/grpc/python/grpc.html#multi-callable-interfaces)) . This requires using the `grpc-stubs` typings for grpcio. This change should allow calling stub methods with common parameters (`timeout`, `metadata`, etc.) as well as calling methods on the `MultiCallable` object (e.g. `my_stub.MyRpcMethod.future()`). 
```

Comment:
@nipunn1313 are you ok that grpc-stubs will be a strict dependency? BTW, this need to be specified as requirement.

Reply:
Ah.. I've forgot that it already is a dependency. Ok let's wait until we can use official master.